src/app.js

```diff
@@ -1,27 +1,78 @@
-import React from 'react';
-import { BrowserRouter } from 'react-router-dom';
+import React, { useEffect } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
 import GlobalStyle from './styles';
 import { Header, Main, ErrorBoundary } from './containers';
-import { Footer } from './components';
+import { Footer, NotFound } from './components';
+import { useDispatch } from 'react-redux';
+import { resetMovie, resetSearch } from './store/actions/movie';
 
 const AppStyled = styled.div`
   display: flex;
   flex-direction: column;
 `;
 
+const Search = ({ search }) => {
```

Comment:
Please move it from app.js

for example, create a directory e.g. /pages and place Search and Movie there